setup.cfg

```diff
@@ -172,3 +172,4 @@ exclude =
 show-source = True
 # E265 - block comment should start with '# '
 extend-ignore = E265
+max-line-length=120
```

Comment:
Yapf ignores this. You'll need to set `COLUMN_LIMIT` in a `.style.yapf`.

Reply:
I'm aware, this is just to allow future contributions to PyInstaller to have longer line limits.